ui/src/utils/DateUtils.ts

```diff
@@ -55,9 +55,9 @@ export const dateTimeString = (date: Date) => {
 export const monthDay = (date: Date) => {
   const hour = addZero(date.getHours());
   const minute = addZero(date.getMinutes());
-  const m = date.getMonth()
+  const m = date.getMonth();
```

Comment:
FIXME: month should use addZero as well but I don't want to muck too much right now and it won't matter until Ocotober (and I hope it doesn't matter at that point)